contracts/tec/contracts/src/libs/LibEIP712Domain.sol

```diff
@@ -0,0 +1,87 @@
+/*
+
+  Copyright 2018 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.2;
+
+
+contract LibEIP712Domain {
```

Comment:
Why use `EIP712` here? TEC's are the ones signing these orders, and they won't be doing it through a signer UI.

Reply:
True... but what's the downside of using EIP712 here? I feel like if we want to push for it as a standard then we should use it for all of our hashing.

Reply:
Fair. Let's keep it.